gradle-plugin-code/src/main/java/com/trivago/cluecumber/logging/GradleCluecumberLogger.java

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
We have a lot of code duplication here. In my opinion, we should have one codebase that is reused as a jar in both implementations.

Reply:
The main thing done here is that we are using different loggers to Gradle and maven tasks. The code I could reuse was moved to the BaseLogger. In order to improve the organization, I'm renaming the class previously called CluecumberLogger to MavenCluecumberLogger and moving it from the Core to the Maven plugin code.